client/src/main/scripts/components/__snapshots__/QuickShare.spec.tsx.snap

```diff
@@ -27,7 +27,7 @@ exports[`It renders 1`] = `
       className="quickshare__url"
     >
       <code>
-        about:///anonymous/untitled/v1/eyJ0aXRsZSI6InVudGl0bGVkIiwiZG9jSWQiOiJpRWZ3SXhNN01tbk9LYjd6dDRIcVc4SXhVV3k2RjdhMjM2ZlNPUWxVVVdJPSIsImhTdGFydCI6IjAuMTAwIiwiaEVuZCI6IjUuMDAwIn0=
+        about:///anonymous/untitled/v1/eyJ0aXRsZSI6InVudGl0bGVkIiwiZG9jSWQiOiJpRWZ3SXhNN01tbk9LYjd6dDRIcVc4SXhVV3k2RjdhMjM2ZlNPUWxVVVdJPSIsImhTdGFydCI6IjAiLCJoRW5kIjoiNSJ9
```

Comment:
Base64 decoding this gives:

```patch
- {"title":"untitled","docId":"iEfwIxM7MmnOKb7zt4HqW8IxUWy6F7a236fSOQlUUWI=","hStart":"0","hEnd":"5"}
+ {"title":"untitled","docId":"iEfwIxM7MmnOKb7zt4HqW8IxUWy6F7a236fSOQlUUWI=","hStart":"0.100","hEnd":"5.000"}
```

which lines up nicely with the change.  LGTM!